startup/gui/usd.py

```diff
@@ -45,3 +45,7 @@
 # `texture:format == automatic` isn't well supported at present, so default
 # user-created lights to `latlong`.
 Gaffer.Metadata.registerValue( GafferUSD.USDLight, "parameters.texture:format", "userDefault", "latlong" )
+
+# Put Arnold parameters last. We can't do that using the schemas in GafferArnold.usda
+# because they provide no control over property ordering.
+Gaffer.Metadata.registerValue( GafferUSD.USDLight, "parameters.arnold:*", "layout:index", -1 )
```

Comment:
I did that hackily in https://github.com/GafferHQ/gaffer/pull/5801/commits/d385bb1fb712cc7091e35fca3d480089e66a35c6, after spending a good amount of time convincing myself that yes, USD was going to return things in a random order, no matter what I did. We might want to revisit when the next renderer gets the UsdLux extension treatment.

Reply:
Thanks, though I'm having trouble locating the reordering commit. The one linked is the portal_mode displayGroup fixup and I'm not seeing anything else in the latest push. I also double checked the branch on your fork to rule out any GitHub weirdness (or at least one level of weirdness).

Reply:
Doh. It was Haddon rather than GitHub weirdness - I had failed to push the final commit. It's there in https://github.com/GafferHQ/gaffer/pull/5801/commits/e291cb732d1d63691056c810a7bedb65c9533544 now.

Reply:
Thanks! My very last pedantic comment would be that having `max_bounces` in the middle of the of the other `Refine` parameters feels a bit odd, though I imagine it's there because `camera` and `transmission` are shuffled to the end because they are not present for all light types?

As an option, here's an alternate ordering roughly based on the "usual" Arnold presentation.
```
"aov", "aov_indirect", "portal_mode", "spread", "roundness", "soft_edge", "camera",
"transmission", "sss", "indirect", "volume", "max_bounces", "cast_volumetric_shadows",
"samples", "volume_samples", "resolution"
```